### DISCUSSION TITLE
Category of Graffiti Sprays capsules

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#117] [General]
Sprays would have capsules for each of them, collection.

![image](https://github.com/user-attachments/assets/51db1a3e-d5ae-4b93-a2d2-c40362d75768)

Checking the list of game items: https://raw.githubusercontent.com/ianlucas/cslib/main/assets/data/items.json

We found all sprays, but I couldn't find the name of their capsules, would there be something like that in the game?

In this case, I would like to leave some sprays deactivated for all users, however putting ID by ID of the graffiti would be more time consuming.

**Comment by ianlucas** [comment]
Well that's actually the container they're coming from, so we could add a rule for not allowing adding an item from a specific container.

**Reply by crashzk** [reply]
It would be perfect, I'll open a REQ/RULE, thanks.
